app/src/main/kotlin/com/wire/android/ui/home/newconversation/search/SearchPeopleScreen.kt

```diff
@@ -98,6 +102,7 @@ private fun SearchResult(
                 contactSearchResult = knownContactSearchResult,
                 showAllItems = searchPeopleScreenState.contactsAllResultsCollapsed,
                 onShowAllButtonClicked = { searchPeopleScreenState.toggleShowAllContactsResult() },
+                invalidQueryMessage = UIText.StringResource(R.string.label_no_results_found).asString(context),
```

Comment:
what is this UIText doing ? isnt it same as stringResource(R.string.label_no_results_found) ?

Reply:
oh yeah true I can use that